File: templates/Uwp/_comp/_shared/Page.TwoPaneView.Common._VB/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.vb
'{**
' This code block adds the method `GetTwoPaneViewDataAsync()` to the SampleDataService of your project.
'**}
Namespace Services
    Public Module SampleDataService
'^^
'{[{

        ' TODO WTS: Remove this once your TwoPaneView pages are displaying real data.
        Async Function GetTwoPaneViewDataAsync() As Task(Of IEnumerable(Of SampleOrder))
            Await Task.CompletedTask
            Return AllOrders()
        End Function
'}]}
    End Module
End Namespace

File: templates/Uwp/_comp/_shared/Page.ListDetail.SampleData/Param_ProjectName.Core/Services/SampleDataService$wts.ItemName_gpostaction.cs
//{**
// This code block adds the method `GetListDetailDataAsync()` to the SampleDataService of your project.
//**}
namespace Param_RootNamespace.Core.Services
{
    public static class SampleDataService
    {
//^^
//{[{

        // TODO WTS: Remove this once your ListDetail pages are displaying real data.
        public static async Task<IEnumerable<SampleOrder>> GetListDetailDataAsync()
        {
            await Task.CompletedTask;
            return AllOrders();
        }
//}]}
    }
}
